### DISCUSSION TITLE
"Push through" mission control when moving a window with "cmd" + mouse

### DISCUSSION THREAD

**Discussion by venpopov** [rxhanson/RectanglePro-Community#597] [Ideas]
If you drag a window the native way by click+drag topbar to the top of the screen, the mission control window activates and you can drop the window in another space. When you use (my favorite Rectangle feature) the `cmd` + mouse focus move, you can move a window, but if you reach the top of the window you cannot push through and activate mission control. Is there any way to make that behavior possible?

https://github.com/user-attachments/assets/34efceb3-5218-4938-9cd8-afffb1176d17

**Comment by rxhanson** [comment]
As a 3rd party app, Rectangle Pro is only able to move/resize windows via the Accessibility API, and unfortunately this cannot be achieved in this way.

This could potentially be achieved by using the Accessibility API to click the window's title bar and drag it when you move the cursor, but that would likely be an undesirable experience. I'll give this some more thought, though.
